lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -742,6 +743,22 @@ public void removeAllAnimatorListeners() {
     animator.removeAllListeners();
   }
 
+  public void addAnimatorPauseListener(Animator.AnimatorPauseListener listener) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
@heitorcolangelo We should add the same `@RequiresApi` here too. This is also a public API.

Reply:
@gpeal
 And do you think we can remove this if statement? I believe it is not necessary as we added the annotation.
The final version would be just
```kotlin
@RequiresApi(api = Build.VERSION_CODES.KITKAT)
public void addAnimatorPauseListener(Animator.AnimatorPauseListener listener) {
  animator.addPauseListener(listener);
}

@RequiresApi(api = Build.VERSION_CODES.KITKAT)
public void removeAnimatorPauseListener(Animator.AnimatorPauseListener listener) {
  animator.removePauseListener(listener);
}
```